tools/purge_omsagent.sh

```diff
@@ -0,0 +1,407 @@
+#! /bin/bash
+
+SOLO_LAD_DIR=/var/lib/waagent/Microsoft.Azure.Diagnostics.LinuxDiagnostic-*
+OMS_LAD_DIR=/var/opt/microsoft/omsagent/LAD
+
+OMS_ONBOARD_AGENT=onboard_agent.sh
```

Comment:
I wanted to make it so if there was firewall issues, getting the onboard_agent.sh script not via a `wget` and then placing it in the same folder as the purge script still worked
(Although I see an issue now, since the onboard_agent.sh script uses `wget` anyway, so it would just prolong the inevitable, I'll change it)